lib/jxl/enc_color_management.cc

```diff
@@ -735,9 +733,7 @@ void ColorEncoding::DecideIfWantICC() {
 ColorSpaceTransform::~ColorSpaceTransform() {
 #if !JPEGXL_ENABLE_SKCMS
   std::lock_guard<std::mutex> guard(LcmsMutex());
```

Comment:
I think this wouldn't be needed anymore, either.

Reply:
I am not entirely sure but it seems to me that it might, based on the comment next to the definition of `LcmsMutex`.

Reply:
Ah, that's disappointing - we should probably not create and/or use it when using skcms though... But that's better left to a separate MR.

Reply:
fwiw, I think that mutex can be removed entirely after commit https://github.com/mm2/Little-CMS/commit/a35bacde52e06aac4704db3e35ad434822938dc2. See https://github.com/mm2/Little-CMS/issues/170#issuecomment-640677221.